tools/nnyeah/nnyeah/MethodTransformations.cs

```diff
@@ -830,6 +846,17 @@ public class MethodTransformations {
 			new Transformation ("System.UInt32 System.nuint::System.IConvertible.ToUInt32(System.IFormatProvider)", ConvertibleMessage),
 			new Transformation ("System.UInt64 System.nuint::System.IConvertible.ToUInt64(System.IFormatProvider)", ConvertibleMessage),
 			new Transformation ("System.Object System.nuint::System.IConvertible.ToType(System.Type,System.IFormatProvider)", ConvertibleMessage),
+
+			// why warnings for this?
+			// Because in .NET 6 there is no direct equivalent of this call.
+			// In order to support this, we need to inject helper methods that do the actual
+			// copying and revector to those helpers.
+			// Not doing this right now since a survey of nuget.org shows that these methods
+			// are never called in the wild.
+			new Transformation ("System.Void System.nuint::CopyArray(System.IntPtr,System.nuint[],System.Int32,System.Int32)",
+				"CopyArray for nuint is not supported yet. If this method is called, it will not function correctly."),
+			new Transformation ("System.Void System.nuint::CopyArray(System.nuint[],System.Int32,System.IntPtr,System.Int32)",
+				"CopyArray for nuint is not supported yet. If this method is called, it will not function correctly."),
```

Comment:
Same message, we might want to use a const variable. Put all together and in the future get the loc team to translate them. We have to make sure that errors are translated to be inclusive.

Reply:
Probably @tj-devel709 can help with this too!